packages/service/schema.json

```diff
@@ -180,6 +180,76 @@
                     }
                   },
                   "additionalProperties": false
+                },
+                "formatters": {
+                  "type": "object",
+                  "properties": {
+                    "bindings": {
+                      "type": "object",
+                      "properties": {
+                        "path": {
+                          "type": "string",
+                          "resolvePath": true
+                        }
+                      },
+                      "additionalProperties": false
+                    },
+                    "level": {
+                      "type": "object",
+                      "properties": {
+                        "path": {
+                          "type": "string",
+                          "resolvePath": true
+                        }
+                      },
+                      "additionalProperties": false
+                    }
+                  },
+                  "additionalProperties": false
+                },
+                "timestamp": {
+                  "anyOf": [
+                    {
+                      "type": "boolean"
+                    },
+                    {
+                      "enum": [
+                        "epochTime",
+                        "unixTime",
+                        "nullTime",
+                        "isoTime"
+                      ]
+                    },
+                    {
+                      "type": "object",
+                      "properties": {
+                        "path": {
+                          "type": "string",
+                          "resolvePath": true
+                        }
+                      },
+                      "additionalProperties": false
+                    }
+                  ]
+                },
+                "redact": {
+                  "type": "object",
+                  "properties": {
+                    "paths": {
+                      "type": "array",
+                      "items": {
+                        "type": "string"
+                      }
+                    },
+                    "censor": {
+                      "type": "string",
+                      "default": "[redacted]"
+                    }
+                  },
+                  "required": [
+                    "paths"
+                  ],
+                  "additionalProperties": false
```

Comment:
I don't see the relative implementation to match this. This doesn't inherit from Basic.

Reply:
I've reverted this